prsa/utilities.py

```diff
@@ -0,0 +1,49 @@
+import numpy as np
+import pandas as pd
+
+def convert_to_time_intervals(anchor_indices: np.ndarray) -> np.ndarray:
+    """
+    Converts anchor point indices to time intervals.
+
+    Parameters:
+    anchor_indices (np.ndarray): Indices of anchor points in the signal.
+
+    Returns:
+    np.ndarray: Time intervals of each anchor point in milliseconds, with dtype adjusted.
+    """
+    return (anchor_indices * 5).astype(np.float64)
```

Comment:
why multiply by 5? please explain - because of our frequency rate? not clear for me what happened here